downgrade

```diff
@@ -135,16 +145,18 @@ declare -a terms
 declare -a to_ignore
 declare -a to_install
 declare -a pacman_args
+declare -a version
 
 while [[ -n "$1" ]]; do
   case "$1" in
     --) shift; pacman_args=( "$@" ); break ;;
+    -v) shift; version=( "$1" ); break ;;
```

Comment:
Yes, in this version I assume that version couldn't be an array but if the pull request is accepted I can work on a version where each version corresponds to the corresponding package in the array. This works now basically to automate the process of downgrading a package when you know the specific version you want.
And you are right again about the break, bug...